apps/backoffice-v2/src/pages/Statistics/components/PortfolioRiskStats/PortfolioRiskStats.tsx

```diff
@@ -0,0 +1,332 @@
+import React, { FunctionComponent, useCallback, useMemo, useState } from 'react';
+import { useAutoAnimate } from '@formkit/auto-animate/react';
+import { Card } from '@/common/components/atoms/Card/Card';
+import { CardHeader } from '@/common/components/atoms/Card/Card.Header';
+import { CardContent } from '@/common/components/atoms/Card/Card.Content';
+import { Cell, Pie, PieChart } from 'recharts';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { Button } from '@/common/components/atoms/Button/Button';
+import { TrendingDown, TrendingUp } from 'lucide-react';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/common/components/atoms/Table';
+import { titleCase } from 'string-ts';
+
+export const PortfolioRiskStats: FunctionComponent = () => {
+  const [parent] = useAutoAnimate<HTMLTableSectionElement>();
+  const [sorting, setSorting] = useState<'asc' | 'desc'>('desc');
```

Comment:
We use `'asc' | 'desc'` type in many places. Maybe we should have a generic `SortDirection` type?